### PR TITLE
#2889 added quick-start tutorial to documentation

### LINKED ISSUE #2889: Synchronize VOLTTRON readme and readthedocs
### Description of Issue

There exist inconsistencies between the main documentation in readthedocs and the readme on github.

Readthedoc main page and readme of github should contain almost all of the same content.  Exceptions are:

  - readthedocs should have a link to a secondary page for rabbitmq installation
  - README should reference the readthedocs secondary page for the main site.
 
Updates to the contents:

 - How to set up the VOLTTRON_HOME variable
 - Show how to tell the agent status ```vctl status```

### PR DESCRIPTION
**Pull request by kefeimo**
# Description

Please include a summary of the change and which issue is fixed
- Added `quick-start.rst`
- Modified `index.rst` to reflect the change in content structure.

Please also include relevant motivation and context.
- Provide an easy-to-follow tutorial for first-time users.

List any dependencies that are required for this change.
- None

Fixes #2889 (Documentation update) 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Content review
- [x] Rendering test

1. `index.rst` rendering, (readthedocs entry): https://volttron-km-readthedocs.readthedocs.io/en/latest/
Picture1

2. `quick-start.rst` rendering, (core content):
https://volttron-km-readthedocs.readthedocs.io/en/latest/developing-volttron/quick-start.html
image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/source/developing-volttron/quick-start.rst [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+After bootstrap finished, we activate the Python virtual environment with the following command:
+
+.. code-block:: bash
+
+       $ source env/bin/activate
+
+You may notice the command prompt has changed and there is the virtual environment name as prefix. e.g., :code:`(volttron) user@host:~/volttron $`. The prefix environment name indicates the virtual environment is activated.
+
+Alternatively, you can use the following command to verify if the virtual environment is up.
```
Comment (craig8) [inline docs/source/developing-volttron/quick-start.rst:148]: This assumes that they don't have a different virtual environment setup such as if you have a conda environment then you already have a virtual env setup.

File: docs/source/developing-volttron/quick-start.rst [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+
+To deactivate the virtual environment (if you run this command, remember to activate the virtual environment again to follow the rest of the steps.)              
+
+.. code-block:: bash
+
+       # Uncomment, if you run this command,
+       # remember to activate the virtual environment again
+       # to follow the rest of the steps
+
+       # deactivate volttron
```
Comment (craig8) [inline docs/source/developing-volttron/quick-start.rst:167]: $ rather than # you can't run as root user.

**Review by craig8 — CHANGES_REQUESTED** [review]
I like what you have done here.

**Comment by craig8** [comment]
We need to update the README for the volttron repository to move the rmq to a link and perhaps link to the quick start tutorial but still include the basic things in the readme.

File: docs/source/developing-volttron/quick-start.rst (hunk 1 above)
Comment (kefeimo) [reply docs/source/developing-volttron/quick-start.rst:148]: Resolve: Added the following note-block
image

File: docs/source/developing-volttron/quick-start.rst (hunk 2 above)
Comment (kefeimo) [reply docs/source/developing-volttron/quick-start.rst:167]: (Originally, I used `#` to indicate the comment-out, but I can see the representation is confusing.)
Resolve with the following modification:
```
Use `deactivate` command to deactivate the virtual environment, i.e., :code:`$ deactivate volttron`. Note: if you run this command, remember to re-activate the virtual environment to follow the rest of the steps.         
```   
image

**Comment by kefeimo** [comment]
Changes:
1. Moved quick-start a new folder called "Tutorials"
image

2. Changed narratives to avoid confusion. (revision as follows.)
image

3. Added note-block to make explicit assumption that the tutorial is under `virtualenv` (instead of, say, `conda`).
image

**Comment by awoz** [comment]
Consider adding a short Deployment section or Next Steps link to "Setting up VOLTTRON as a System Service". 
I found that after learning the Quick Start operations, a developer would want to auto-run the VOLTTRON platform and demo agents.

**Pull request merged**